targets/tizenrt-artik053/apps/jerryscript/jerry_main.c

```diff
@@ -482,6 +482,21 @@ jerry_cmd_main (int argc, char *argv[])
   return ret_code;
 } /* jerry_cmd_main */
 
+/**
+ * Run JerryScript and print its return value.
+ */
+static int
+jerry(int argc, char *argv[])
+{
+  int ret_code = jerry_cmd_main(argc, argv);
+
+#ifdef CONFIG_DEBUG_VERBOSE
+  printf("JerryScript Result: %d\n", ret_code);
```

Comment:
lowercase "result"

Reply:
@rtakacs What about [jerry_port_log](https://github.com/jerryscript-project/jerryscript/blob/master/targets/tizenrt-artik053/apps/jerryscript/jerry_main.c#L497), insted of `printf`?

Reply:
@glistening Good idea, thanks.
@zherczeg Fixed.